### PR TITLE
feat: rename inputs and add them to ibm_catalog.json

### PR DESCRIPTION
**Pull request by akocbek**
### Description

Issue:
https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/issues/36 papercut: missing rich variable types 
https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/issues/39 papercut: expose full product name in variables 
https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/issues/35 papercut: group related inputs together in project UI

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@akocbekIBM can you change back the indentation in the json to 2 spaces instead of tabs please?

File: ibm_catalog.json [hunk 1]
```diff
@@ -1,117 +1,345 @@
...
+                      "key": "cos_region",
+                      "options": [
                         {
-                            "service_name": "compliance",
-                            "role_crns": [
-                                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
-                                "crn:v1:bluemix:public:iam::::role:Editor"
-                            ]
+                          "displayname": "Dallas (us-south)",
+                          "value": "us-south"
```
Comment (ocofaigh) [inline ibm_catalog.json:102]: use the predefined region picker for COS:
```
{
  "custom_config": {
    "config_constraints": {
      "generationType": "2"
    },
    "grouping": "deployment",
    "original_grouping": "deployment",
    "type": "region"
  },
  "key": "cos_region",
  "required": true,
  "type": "string"
}
```

**Comment by akocbek** [comment]
created new PR with correct spacing https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/pull/61. Closing this one

**Pull request closed without merging**